Lib/pickletools.py

```diff
@@ -1228,8 +1228,7 @@ def __init__(self, name, code, arg,
 
       The same as INT, except that the literal ends with 'L', and always
       unpickles to a Python long.  There doesn't seem a real purpose to the
-      trailing 'L', and the trailing 'L' is not required for Python 3.0 or
-      higher.
+      trailing 'L' and it's not required for Python 3.0 or higher.
```

Comment:
Preserve the Oxford comma
```suggestion
      trailing 'L', and it's not required for Python 3.0 or higher.
```